webapp/src/components/MeetingAccordion.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Link as MuiLink,
+  ListItem,
+  ListItemText,
+} from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Link as ReactRouterLink } from 'react-router-dom';
+import React from 'react';
+
+import { formatDate, formatTime } from '../helpers/dateTime';
+import { Meeting as MeetingType } from '../types/api';
```

Comment:
I've seen other files that alias api types to `APIMeeting`, and things like that. Would it be valuable to use that convention consistently?

Reply:
Yes, that would be valuable.